packages/discussions/graphql/resolvers/Discussion/userCanComment.js

```diff
@@ -1,5 +1,14 @@
 const { Roles } = require('@orbiting/backend-modules-auth')
 
-module.exports = async ({ minInterval, id }, _, { pgdb, user }) => {
-  return Roles.userIsInRoles(user, ['member'])
+const {
+  activeMembership: getActiveMembership
+} = require('../../../../../servers/republik/modules/crowdfundings/graphql/resolvers/User')
+
+module.exports = async ({ minInterval, id }, _, context) => {
+  const { user } = context
+
+  const isMember = Roles.userIsInRoles(user, ['member'])
+  const hasActiveMembership = !!(await getActiveMembership(user, null, context))
```

Comment:
why is `isMember` not enough?

Reply:
As of this Pull Request, you will require an active membership to comment. Just a `member` role, assigned when claiming an access grant, won't be enough anymore.